src/error.rs

```diff
@@ -157,19 +157,19 @@ impl fmt::Display for Error {
         match self {
             Error::ArraySizeZero => write!(
                 f,
-                "Array size must be positive: 1, 2, 3, 4, 5, ..."
+                "Array size must be positive: 1, 2, 3, 4, 5, ... Got 0"
```

Comment:
In 23611c2b3757ca5ac0188b72ddac700b2b87e9f7:

Since I am nitting grammer, `got` should be lowercase in all these.